<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Embed</class>
 <widget class="QDialog" name="Embed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Embed</string>
  </property>
  <widget class="QFrame" name="embed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <widget class="QPushButton" name="browseAudio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Text Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>or</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Text File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseFile">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="embedButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Embed</string>
    </property>
   </widget>
   <widget class="QLabel" name="textFilenameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="audioFilenameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="messageBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>177</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File Name</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="filenameBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the Key</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="keyBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EMBEDDING WINDOW</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFrame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/QFrame</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
